verilog/analysis/verilog_linter_configuration.h

```diff
@@ -130,6 +130,30 @@ struct ProjectPolicy {
   std::string ListPathGlobs() const;
 };
 
+struct LinterConfigurationFlags {
+  // strings, ints, bools, and unprocessed values from flags, no other derived
+  // information. Reasonable default values may be specified here for each
+  // member
+
+  // The base set of rules used by linter
+  const RuleSet& ruleset;
```

Comment:
This essentially is a pointer to a rule-set and given that the struct is meant to be passed around self-contained, I think this should be a  copy of a rule-set. 

If this is not meant to be an instance but in fact a reference (e.g. because you want to build trees of rules-sets), I'd make it a const pointer, so that the lifetime-implications are very clear (and they shoudl also be mentioned in the field documentation).

Reply:
Sure, thanks, changed to a copy